tests/libvmi/operations.go

```diff
@@ -0,0 +1,31 @@
+package libvmi
+
+import (
+	. "github.com/onsi/gomega"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/tests/framework/matcher"
+)
+
+func CreateVmi(client kubecli.KubevirtClient, namespace string, vmi *v1.VirtualMachineInstance) (*v1.VirtualMachineInstance, error) {
+	vmi, err := client.VirtualMachineInstance(namespace).Create(vmi)
+	if err != nil {
+		return nil, err
+	}
+	return vmi, nil
+}
+
+func DeleteVmi(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) error {
+	return virtClient.VirtualMachineInstance(vmi.Namespace).Delete(vmi.Name, &v12.DeleteOptions{})
+}
+
+func WaitForVmiDeletion(vmi *v1.VirtualMachineInstance) {
+	EventuallyWithOffset(1, matcher.ThisVMI(vmi), 120).Should(matcher.BeGone())
+}
+
+func DeleteVmiAndWaitForDeletion(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) {
+	Expect(DeleteVmi(virtClient, vmi)).To(Succeed())
```

Comment:
Same, lets not assert here. Just return an error and assert at the test itself.

Reply:
Once we will decide on this thread what should be done, I'll act accordingly. Originally I returned an error and did not assert but @rmohr asked me to assert.